src/components/views/Game.js

```diff
@@ -123,6 +123,10 @@ export default function Game() {
                 await new Promise(resolve => setTimeout(resolve, 100));
                 setRounds(responseGame.data.settings.rounds);
                 setRoundsPlayed(responseGame.data.roundsPlayed);
+                setTeam1Size(responseGame.data.team1.players.length);
+                setTeam2Size(responseGame.data.team2.players.length);
+                console.log(team1Size);
+                console.log(team2Size);
```

Comment:
Add a string to the log that informs us what kind of information is being logged. I.e. "Team size: ".